app-dicts/myspell-lv/myspell-lv-1.3.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+MYSPELL_DICT=(
+        "lv_LV.aff"
```

Comment:
Thanks. Still I have no idea how to configure tabs and use repoman via github web interface. Some hints would be helpful.